tm2/pkg/crypto/keys/types.go

```diff
@@ -44,20 +44,15 @@ type Keybase interface {
 	// CreateMulti creates, stores, and returns a new multsig (offline) key reference
 	CreateMulti(name string, pubkey crypto.PubKey) (info Info, err error)
 
-	// The following operations will *only* work on locally-stored keys
-	// In all import operations, if an account exists with the same address but a different name, it is replaced by the new name.
+	// Rotate replaces the encryption password for a given key
 	Rotate(name, oldpass string, getNewpass func() (string, error)) error
-	Import(name string, armor string) (err error)
-	ImportPrivKey(name, armor, decryptPassphrase, encryptPassphrase string) error
-	ImportPrivKeyUnsafe(name, armor, encryptPassphrase string) error
-	ImportPubKey(name string, armor string) (err error)
-	Export(name string) (armor string, err error)
-	ExportPubKey(name string) (armor string, err error)
-	ExportPrivKey(name, decryptPassphrase, encryptPassphrase string) (armor string, err error)
-	ExportPrivKeyUnsafe(name, decryptPassphrase string) (armor string, err error)
-
-	// ExportPrivateKeyObject *only* works on locally-stored keys. Temporary method until we redo the exporting API
-	ExportPrivateKeyObject(name string, passphrase string) (crypto.PrivKey, error)
+
+	// ImportPrivKey imports the given private key into the keybase.
+	// In all import operations, if an account exists with the same address but a different name, it is replaced by the new name
+	ImportPrivKey(name string, key crypto.PrivKey, encryptPass string) error
+
+	// ExportPrivKey exports the private key from the keybase. It *only* works on locally-stored keys
+	ExportPrivKey(name string, decryptPass string) (crypto.PrivKey, error)
```

Comment:
```suggestion
	ExportPrivKey(name string, decryptPass, encryptPass string) (crypto.PrivKey, error)
```

Reply:
The intention of this PR was to remove the moment where the keybase API has to manage armor and application-level formats (such as encrypted / unencrypted armor).

Locally, the keybase uses armor to save, but the API should not have this limitation, hence why `encryptPass` was dropped